showcase/src/app/demo-ui/pages/date-range-picker/demo-date-range-picker-page.component.html

```diff
@@ -1,80 +1,57 @@
 <h1 class="mat-display-3" translate>SHOWCASE.DEMO.DATE_RANGE_PICKER.TITLE</h1>
 <section class="stark-section">
 	<h1 translate>SHOWCASE.DEMO.SHARED.EXAMPLE_VIEWER_LIST</h1>
-	<example-viewer [extensions]="['HTML', 'TS']" filesPath="date-range-picker/full" exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.TITLE">
-		<div>
-			<stark-date-range-picker
-				rangePickerId="id-range-picker"
-				rangePickerName="name-range-picker"
-				[isDisabled]="disabled"
-				[dateFilter]="customDateFilter"
-				[startDate]="startDate"
-				startDateLabel="STARK.DATE_RANGE_PICKER.FROM"
-				[endDate]="endDate"
-				endDateLabel="STARK.DATE_RANGE_PICKER.TO"
-				[startMinDate]="minDate"
-				[startMaxDate]="maxDate"
-				[endMinDate]="startDate || minDate"
-				[endMaxDate]="maxDate"
-				(dateRangeChanged)="onDateChanged($event)"
-			>
-			</stark-date-range-picker>
-		</div>
-		<mat-checkbox [(ngModel)]="disabled">isDisabled</mat-checkbox>
+	<example-viewer
+		[extensions]="['HTML', 'TS']"
+		filesPath="date-range-picker/model"
+		exampleTitle="SHOWCASE.DEMO.DATE_RANGE_PICKER.WITH_MODEL"
+	>
+		<stark-date-range-picker
+			[(ngModel)]="dateRangeModel"
+			(dateRangeChanged)="onDateModelChange()"
+			dateFilter="OnlyWeekdays"
+			[startMinDate]="today"
+			[endMinDate]="today"
+			[startMaxDate]="inOneMonth"
+			[endMaxDate]="inOneMonth"
+			[disabled]="modelDisabled"
+		></stark-date-range-picker>
+		<br />
+		<mat-checkbox [(ngModel)]="modelDisabled">
+			{{ (modelDisabled ? "SHOWCASE.COMMON.DISABLED" : "SHOWCASE.COMMON.ENABLED") | translate }}
```

Comment:
Can you add the same functionality for the other example with FormGroup? In that case it is just about setting the formGroup as disabled right?